src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -2455,3 +2455,40 @@ function vertex_figure(P::Polyhedron{T}, n::Int; cutoff=nothing) where {T<:scala
     Polymake.polytope.vertex_figure(pm_object(P), n - 1; opts...), coefficient_field(P)
   )
 end
+
+@doc raw"""
+    tutte_lifting(G::Graph{Undirected})
+
+Compute a realization of `G` in $\mathbb{R}^3$, i.e., a polyhedron whose edge graph is `G`.  Assumes that `G` is planar, 3-connected, and that is has a triangular facet.
+
+# Examples
+```jldoctest
+julia> G = edgegraph(simplex(3))
```

Comment:
```suggestion
julia> G = vertex_edge_graph(simplex(3))
```
We do have an alias for `edgegraph`, but I think `vertex_edge_graph` is the recommended name to use.